packages/v4/patternfly-docs/content/design-guidelines/components/form/forms.md

```diff
@@ -113,7 +113,9 @@ Form submit buttons should be disabled until a user starts filling out the form
 * For a submitted form, disable the submit button while a user's responses are processing to avoid a duplicate submission.
```

Comment:
The "Content" link about doesn't work because it needs to be updated to match the new "Content considerations" header.

So should be (#content-considerations)

Reply:
@wolfeallison all the content links aren't working weirdly... not sure why?? In this case you have 2 hashtags instead of 1, so that's an easy fix, but if you do control+F on the rest of them, i'm not sure what the problem is hmmm